appengine/genetics/style.css

```diff
@@ -0,0 +1,171 @@
+#blockly {
+    position: fixed;
+    bottom: 10px;
+}
+
+#editor {
+    font-size: 16pt;
+    position: fixed !important;
```

Comment:
I think it's generally a good idea to avoid "!important" where possible, since this overrides the usual order of precedence. Is there a reason you need it? I checked a couple of the other games, and it didn't seem to be needed there.